<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChargingReport</class>
 <widget class="QWidget" name="ChargingReport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinAndMaxSize</enum>
   </property>
   <property name="leftMargin">
    <number>40</number>
   </property>
   <property name="topMargin">
    <number>30</number>
   </property>
   <property name="rightMargin">
    <number>40</number>
   </property>
   <property name="bottomMargin">
    <number>35</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: \*url();

border-image: url(:/background1.png);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="1,3,1,6,1,1">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinAndMaxSize</enum>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="9,1">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinAndMaxSize</enum>
          </property>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>578</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="TimeLimit" name="timeLimit" native="true"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,1,1">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinAndMaxSize</enum>
          </property>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>147</width>
              <height>110</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>150</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>150</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/battery.png);</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QProgressBar" name="progressBarSoc">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>84</width>
                 <height>125</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>84</width>
                 <height>120</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">border-image: \*url();
font: 19pt &quot;Sans Serif&quot;;
selection-background-color: rgb(0, 255, 0);

background-color: rgb(215, 248, 211);</string>
               </property>
               <property name="value">
                <number>0</number>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="textDirection">
                <enum>QProgressBar::TopToBottom</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>147</width>
              <height>110</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>17</width>
            <height>13</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="labelChargingEnergy1">
            <property name="font">
             <font>
              <family>Sans Serif</family>
              <pointsize>20</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 20pt &quot;Sans Serif&quot;;
color: rgb(0, 0, 0);
border-image: \*url();
</string>
            </property>
            <property name="text">
             <string>充电电量(kWh):</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="labelChargingEnergy">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Sans Serif</family>
              <pointsize>17</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
              <underline>true</underline>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
font: 17pt &quot;Sans Serif&quot;;
text-decoration: underline;
border-image: \*url();</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="labelVoltage1">
            <property name="font">
             <font>
              <family>Sans Serif</family>
              <pointsize>20</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 20pt &quot;Sans Serif&quot;;
color: rgb(0, 0, 0);
border-image: \*url();</string>
            </property>
            <property name="text">
             <string>输入电压(V):</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="labelVoltage">
            <property name="font">
             <font>
              <family>Sans Serif</family>
              <pointsize>17</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
              <underline>true</underline>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
font: 17pt &quot;Sans Serif&quot;;
text-decoration: underline;
border-image: \*url();</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelSumEnergy1">
            <property name="font">
             <font>
              <family>Sans Serif</family>
              <pointsize>20</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 20pt &quot;Sans Serif&quot;;
color: rgb(0, 0, 0);
border-image: \*url();</string>
            </property>
            <property name="text">
             <string>总  电  能(kWh):</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="labelSumEnergy">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Sans Serif</family>
              <pointsize>17</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
              <underline>true</underline>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
font: 17pt &quot;Sans Serif&quot;;
text-decoration: underline;
border-image: \*url();
</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelPower1">
            <property name="font">
             <font>
              <family>Sans Serif</family>
              <pointsize>20</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 20pt &quot;Sans Serif&quot;;
color: rgb(0, 0, 0);
border-image: \*url();</string>
            </property>
            <property name="text">
             <string>充电 功率 (kW):</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="labelPower">
            <property name="font">
             <font>
              <family>Sans Serif</family>
              <pointsize>17</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
              <underline>true</underline>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
font: 17pt &quot;Sans Serif&quot;;
text-decoration: underline;
border-image: \*url();</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelChargedTime1">
            <property name="font">
             <font>
              <family>Sans Serif</family>
              <pointsize>20</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 20pt &quot;Sans Serif&quot;;
color: rgb(0, 0, 0);
border-image: \*url();</string>
            </property>
            <property name="text">
             <string>充电用时 (min):</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="labelChargedTime">
            <property name="font">
             <font>
              <family>Sans Serif</family>
              <pointsize>17</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
              <underline>true</underline>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
font: 17pt &quot;Sans Serif&quot;;
text-decoration: underline;
border-image: \*url();
</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="labelDcVoltage1">
            <property name="font">
             <font>
              <family>Sans Serif</family>
              <pointsize>20</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 20pt &quot;Sans Serif&quot;;
color: rgb(0, 0, 0);
border-image: \*url();</string>
            </property>
            <property name="text">
             <string>输出电压(V):</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="labelDcVoltage">
            <property name="font">
             <font>
              <family>Sans Serif</family>
              <pointsize>17</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
              <underline>true</underline>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
font: 17pt &quot;Sans Serif&quot;;
text-decoration: underline;
border-image: \*url();
</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="labelDcElectricity1">
            <property name="font">
             <font>
              <family>Sans Serif</family>
              <pointsize>20</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 20pt &quot;Sans Serif&quot;;
color: rgb(0, 0, 0);
border-image: \*url();</string>
            </property>
            <property name="text">
             <string>输出电流(A):</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLabel" name="labelDcElectricity">
            <property name="font">
             <font>
              <family>Sans Serif</family>
              <pointsize>17</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
              <underline>true</underline>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
font: 17pt &quot;Sans Serif&quot;;
text-decoration: underline;
border-image: \*url();</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="labelStartTime1">
            <property name="font">
             <font>
              <family>Sans Serif</family>
              <pointsize>20</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 20pt &quot;Sans Serif&quot;;
color: rgb(0, 0, 0);
border-image: \*url();</string>
            </property>
            <property name="text">
             <string>充电  开始时间 :</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QLabel" name="labelStartTime">
            <property name="font">
             <font>
              <family>Sans Serif</family>
              <pointsize>17</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
              <underline>true</underline>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
font: 17pt &quot;Sans Serif&quot;;
text-decoration: underline;
border-image: \*url();</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>17</width>
            <height>13</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="6">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinAndMaxSize</enum>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,6,1">
            <property name="sizeConstraint">
             <enum>QLayout::SetMinAndMaxSize</enum>
            </property>
            <item>
             <widget class="QPushButton" name="buttonSave">
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton { background-color: rgb(0, 170, 255); color: rgb(0, 0, 0); } 
QPushButton:hover:pressed{ background-color: rgb(0,0,127); color:rgb(255, 255, 255);} 
QPushButton:hover:!pressed{ background-color: rgb(0, 170, 255); color: rgb(0, 0, 0); }
QPushButton {font: 27pt ; } </string>
              </property>
              <property name="text">
               <string>返回</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>358</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="buttonStopCharging">
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton { background-color: rgb(0, 170, 255); color: rgb(0, 0, 0); } 
QPushButton:hover:pressed{ background-color: rgb(0,0,127); color:rgb(255, 255, 255);} 
QPushButton:hover:!pressed{ background-color: rgb(0, 170, 255); color: rgb(0, 0, 0); }
QPushButton {font: 27pt ; } </string>
              </property>
              <property name="text">
               <string>结束充电</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TimeLimit</class>
   <extends>QWidget</extends>
   <header>TimeLimit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
